CMSgov/design-system
# [WNMGDS-152] Child design system example

### Added
- Adding README to the example explaining how to set up a child design system 
- Added source code to get started

### How to test
- go to the examples > child design system
- run `yarn install`
- run `yarn start` 
- ensure the site is running locally and any changes you make to the following rebuild
    - component changes
    - component style changes
    - documentation changes
    - example code changes
